cmd/qat_plugin/README.md

```diff
@@ -71,29 +82,31 @@ $ kubectl describe node <node name> | grep intel.com/qat
 
 1. Place the Dockerfile in the DPDK directory and build the DPDK image:
 
-   ```
-   $ cd demo
-   $ docker build -t crypto-perf .
-   ```
+     ```
+     $ cd demo
+     $ docker build -t crypto-perf .
+     ```
 
-   This command will produce a Docker image named `ubuntu-demo-opencl`.
+     This command produces a Docker image named `ubuntu-demo-opencl`.
 
-2. Running an example DPDK application (dpdk-test-crypto-perf) requesting QAT devices:
+2. Run an example DPDK application (`dpdk-test-crypto-perf`) requesting QAT devices:
 
-   Add container resource request and limit e.g. **intel.com/qat: '<number of devices>'** for the container requesting quick assist device in the pod specification file. For a DPDK based workload hugepage request and limit might also be needed.
-   ```
-   $ kubectl create -f demo/crypto-perf-dpdk-pod-requesting-qat.yaml
-   $ kubectl get pods
-     NAME                     READY     STATUS    RESTARTS   AGE
-     dpdkqatuio               1/1       Running   0          27m
-     intel-qat-plugin-5zgvb   1/1       Running   0          3h
+      In the pod specification file, add container resource request and limit.
+      For example, `intel.com/qat: <number of devices>` for a container requesting QAT devices.
 
+      For a DPDK based workload, you may need to add hugepage request and limit.
 
```

Comment:
@swatisehgal  Please see if step 2 makes sense - should the user first update the pod spec file and then run the example DPDK application? (I am wondering if lines 94-97 are step 2, then step 2 becomes 3, etc.)

Reply:
yes that's correct the pod specification needs to be updated before running the DPDK application. May be we can change this as follows:
2. Deploy a Pod to run the DPDK application (dpdk-test-crypto-perf)
In the pod specification file, add container resource request and limit. For example, intel.com/qat: <number of devices> for a container requesting QAT devices.

For a DPDK based workload, you may need to add hugepage request and limit.

$ kubectl create -f demo/crypto-perf-dpdk-pod-requesting-qat.yaml
$ kubectl get pods
  NAME                     READY     STATUS    RESTARTS   AGE
  dpdkqatuio               1/1       Running   0          27m
  intel-qat-plugin-5zgvb   1/1       Running   0          3h

3. Manually execute the dpdk-test-crypto-perf application to review the logs. 
$ kubectl exec -it dpdkqatuio bash
$ ./dpdk-test-crypto-perf -l 6-7 -w $intelQAT1 -- --ptest throughput --\
 devtype crypto_qat --optype cipher-only --cipher-algo aes-cbc --cipher-op \
 encrypt --cipher-key-sz 16 --total-ops 10000000 --burst-sz 32 --buffer-sz 64

Reply:
I hope that makes sense, let me know if you have questions :)

Reply:
@swatisehgal  Done, please review the latest.